README.md

```diff
@@ -203,7 +199,7 @@ end
 
 ---------------------------------------------
 
-![Empty Tile Output](https://github.com/alinchis/Escher.jl/blob/master/assets/img/empty_tile.png "Empty Tile")
+![Empty Tile Output](./assets/img/empty_tile.png "Empty Tile")
```

Comment:
use https://cloud.githubusercontent.com/assets/25916/15371154/d15b3bc2-1d57-11e6-8d41-1f2e5d535d78.png this URL, and you can remove the file from the repo